src/Microsoft.ML.Data/Transforms/KeyToValueTransform.cs

```diff
@@ -32,7 +32,7 @@
 [assembly: LoadableClass(typeof(IRowMapper), typeof(KeyToValueTransform), null, typeof(SignatureLoadRowMapper),
     KeyToValueTransform.UserName, KeyToValueTransform.LoaderSignature)]
 
-namespace Microsoft.ML.Runtime.Data
+namespace Microsoft.ML.Transforms.CategoricalTransforms
```

Comment:
>CategoricalTransforms [](start = 34, length = 21)

Do we really want subcategories for transforms?

Reply:
YEs, look at #1318 the last section.